rs/tests/consensus/tecdsa/BUILD.bazel

```diff
@@ -5,6 +5,7 @@ package(default_visibility = ["//rs:system-tests-pkg"])
 
 system_test_nns(
     name = "tecdsa_complaint_test",
+    extra_head_nns_tags = [],  # don't run the head_nns variant on nightly since it aleady runs on slow_test.
```

Comment:
I don't think this test was added to slow_test (unlike the comment states)

Reply:
Well spotted. This test indeed has a P90 of 4m 31s so I'll remove the `extra_head_nns_tags`.